app/views/reports/_team_score_score_awarded.html.erb

```diff
@@ -1,8 +1,10 @@
-<% if !@assignment.varying_rubrics_by_round? %>
+<% unless @assignment.varying_rubrics_by_round? %>
     <!--Score awarded-->
     <div>
-    <% if @review_scores[reviewer_id][team_id] != -1%>
-        <%= @review_scores[reviewer_id][team_id].inspect %>%
+    <% if @review_scores[reviewer_id][team_id] != -1 %>
```

Comment:
i think this should be converted into an unless and put below. just remember boolean algebra :)